lib/options.js

```diff
@@ -32,8 +32,8 @@ module.exports = {
         name: 'Set indent character',
         id: 'indentCharacter',
         type: 'enum',
-        default: ' ',
-        availableOptions: [' ', '\t'],
+        default: 'Space',
+        availableOptions: ['Space', 'Tab'],
```

Comment:
@shreys7 the handling for these needs to be changed as well

Reply:
Already done